base/test-valencycheck/src/test/java/org/openscience/cdk/tools/DeduceBondOrderTestFromExplicitHydrogens.java

```diff
@@ -151,19 +148,19 @@ void xtestButadiene() throws Exception {
         IAtomContainer enol = SilentChemObjectBuilder.getInstance().newAtomContainer();
 
         // atom block
-        IAtom atom1 = new Atom(Elements.CARBON);
```

Comment:
All good but the more efficient and cleaner way to do this is (recent change):

```java
IAtom atom1 = enol.newAtom(Elements.CARBON); // maybe need hydrogen count
...
IBond bond1 = enol.newBond(atom1, atom2);
```

This is also less error prone with the AtomContainer2 which wants to have "owned" atoms. If you create an atom in isolation then creating bonds needs to be done with care - I got most of the use cases but ``mol.newAtom(...)`` is nice an succient.

Reply:
Is the `.newAtom()` method part of the `IAtomContainer` interface? It seems not. Do we want that?

Reply:
Should be... https://github.com/cdk/cdk/blob/main/base/interfaces/src/main/java/org/openscience/cdk/interfaces/IAtomContainer.java#L486

Reply:
Oh, duh, how did I miss that. Something for the groovy book.